generic_tracers/generic_COBALT.F90

```diff
@@ -3514,14 +3514,17 @@ subroutine generic_COBALT_update_from_source(tracer_list,Temp,Salt,rho_dzt,dzt,h
     !
     do k = 1, nk ; do j = jsc, jec ; do i = isc, iec   !{
        do n = 1, NUM_PHYTO  !{
+          ! Take up iron if below maximum guota and day averaged growth is positive
```

Comment:
@charliestock, could you double-check this comment? I might be misunderstanding your code, but it seems like you only have the maximum quota check here, and I didn't see any check for the growth rate.

Reply:
growth check added.  Thanks for catching this.